src/sentry/static/sentry/app/components/u2f/u2finterface.tsx

```diff
@@ -188,7 +208,7 @@ class U2fInterface extends React.Component {
                   support,
                 }
               ),
-            }[deviceFailure]
+            }[deviceFailure || '']
```

Comment:
you don't need `' '` here 😉
```suggestion
            }[deviceFailure]
```

Reply:
It's `string | null`, null can't be used as an index type.

Reply:
oh yeah..typescript :)